primitive-types/src/lib.rs

```diff
@@ -14,216 +14,200 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-#[cfg(feature = "std")]
-extern crate core;
-
-#[macro_use]
-extern crate uint;
-
-#[macro_use]
-extern crate fixed_hash;
-
-#[cfg(feature = "impl-serde")]
-#[macro_use]
-extern crate impl_serde;
-
-#[cfg(feature = "impl-codec")]
-#[macro_use]
-extern crate impl_codec;
-
-#[cfg(feature = "impl-rlp")]
-#[macro_use]
-extern crate impl_rlp;
-
 use core::convert::TryFrom;
+use fixed_hash::*;
+use impl_codec::*;
```

Comment:
Please retain the feature attributes, seems like they got lost.

Reply:
They are feature flags to provide conditional compilation (such as `#[cfg(feature = "impl-codec")]` to include in the build onlu when `feature` == impl-codec is enabled.

For further reading see: https://doc.rust-lang.org/1.0.0/book/conditional-compilation.html